util/fibers/detail/fiber_interface.h

```diff
@@ -184,10 +184,18 @@ class FiberInterface {
   ::boost::context::fiber_context Terminate();
 
   std::atomic<uint32_t> use_count_;  // used for intrusive_ptr refcounting.
-  std::atomic<uint16_t> flags_;
 
+  // trace_ variable - used only for debugging purposes.
+  enum TraceState : uint8_t {
+    TRACE_NONE,
+    TRACE_SLEEP_WAKE,
+    TRACE_TERMINATE,
+    TRACE_READY
+  } trace_ = TRACE_NONE;
```

Comment:
I'm curious: did you intentionally use this syntax, or did you forget the `;` and clang-format merged the lines?

```suggestion
  };
  trace_ = TRACE_NONE;
```

Reply:
that was intentional